br/pkg/task/stream.go

```diff
@@ -1914,7 +1914,7 @@ func parseFullBackupTablesStorage(
 	}, nil
 }
 
-func initRewriteRules(schemasReplace *stream.SchemasReplace) map[int64]*restoreutils.RewriteRules {
```

Comment:
Say... We may register the `shiftedStartTs`, `startTs` and `restoredTs` to each rewrite rules in the returned map.

Reply:
yes, we don't have such common rule for rewrite, it's bind with table.